events/2022.07.Online/TD/flask-tum/manual.csv

```diff
@@ -0,0 +1,109 @@
+"td-context-default-language-direction-independence","null","However, when interpreting human-readable text, each human-readable string value MUST be processed independently."
+"td-default-alg","null","The value associated with member alg if not given MUST be assumed to have the default value ES256."
+"td-default-contentType","null","The value associated with member contentType if not given MUST be assumed to have the default value application/json."
+"td-default-format","null","The value associated with member format if not given MUST be assumed to have the default value jwt."
+"td-default-http-method","null","When no method is indicated in a form representing an Protocol Binding based on HTTP, a Default Value MUST be assumed as shown in the following table."
+"td-default-http-method_get","null","The value associated with member GET if not given MUST be assumed to have the default value  Form with operation type readproperty"
+"td-default-http-method_post","null","The value associated with member POST if not given MUST be assumed to have the default value Form with operation type invokeaction."
+"td-default-http-method_put","null","The value associated with member PUT if not given MUST be assumed to have the default value Form with operation type writeproperty."
+"td-default-idempotent","null","The value associated with member idempotent if not given MUST be assumed to have the default value false."
+"td-default-in-apikey","null","The value associated with member in if not given MUST be assumed to have the default value query."
+"td-default-in-basic","pass","The value associated with member in if not given MUST be assumed to have the default value header."
+"td-default-in-bearer","null","The value associated with member in if not given MUST be assumed to have the default value header."
+"td-default-in-digest","null","The value associated with member in if not given MUST be assumed to have the default value header."
+"td-default-op-actions","pass","The value associated with member op if not given MUST be assumed to have the default value invokeaction."
+"td-default-op-events","pass","The value associated with member op if not given MUST be assumed to have the default value subscribeevent."
+"td-default-op-properties","pass","The value associated with member op if not given MUST be assumed to have the default value Array of string with the elements readproperty and writeproperty."
+"td-default-qop","null","The value associated with member qop if not given MUST be assumed to have the default value auth."
+"td-default-readOnly","pass","The value associated with member readOnly if not given MUST be assumed to have the default value false."
+"td-default-safe","pass","The value associated with member safe if not given MUST be assumed to have the default value false."
+"td-default-success","null","."
+"td-default-writeOnly","pass","The value associated with member writeOnly if not given MUST be assumed to have the default value false."
```

Comment:
actually the default assertions make sense only for consumers since the Things here do not interpret this